db/migrate/20200721230149_create_games.rb

```diff
@@ -0,0 +1,32 @@
+class CreateGames < ActiveRecord::Migration[5.2]
+  def change
+    create_table :games do |t|
+      t.string :title
+      t.integer :age_rating
+      t.float :agg_rating
+      t.text :artworks, array: true, default: []
+      t.integer :category
+      t.string :cover
+      t.integer :follows
+      t.text :game_modes, array: true, default: []
+      t.integer :hypes
+      t.text :companies, array: true, default: []
+      t.text :keywords, array: true, default: []
+      t.text :multiplayer_modes, array: true, default: []
+      t.text :player_perspective, array: true, default: []
+      t.float :popularity
+      t.float :rating
+      t.integer :rating_count
+      t.text :release_date, array: true, default: []
+      t.text :screenshots, array: true, default: []
+      t.text :similar_games, array: true, default: []
+      t.string :storyline
+      t.text :themes, array: true, default: []
+      t.float :total_rating
+      t.string :url
+      t.string :video
```

Comment:
This is far more columns than we discussed but I thought we could remove them if we dont need them

Reply:
We have lots of ideas on what to implement so totally makes sense that we just remove what we don't need.